template/src/pageComponents/buy-me-coffee/Introduction.tsx

```diff
@@ -0,0 +1,27 @@
+import Image from 'next/image';
+import UserAvatar from './Avatar.svg';
+
+export default function Introduction() {
+  const coffeeReceivedCount = 90;
+  const userName = '0xPudgy';
+  const aboutUser = 'I am a content creator producing videos about NFTs!';
+
+  return (
+    <section>
+      <div className="mx-[-100vw] my-10 h-px bg-zinc-400 bg-opacity-20 md:my-16" />
+      <div className="flex flex-col justify-start gap-8 md:flex-row md:items-center">
+        <Image alt="user-avatar" src={UserAvatar} className="h-28 w-28" />
```

Comment:
@arsood I wonder what avatar should we have here. Maybe the person deploying the contract?

Reply:
Yes it should be the avatar of user deploying the contract. I have hard-coded avatar image for now.

Reply:
Yes, agreed on the person who is deploying the contract